core/src/Kokkos_MemoryPool.hpp

```diff
@@ -525,6 +525,10 @@ class MemoryPool {
     // the guess for which block within a superblock should
     // be claimed.  If not available then a search occurs.
 
+// FIXME_SYCL clock_tic not yet implemented
+#ifdef KOKKOS_ENABLE_SYCL
+    const uint32_t block_id_hint = 0;
```

Comment:
Use something like iteration index or whatever here. Using 0 is probably pretty bad performance wise - i mean bad enough that the test will take really long. Or?

Reply:
```
37/41 Test #37: KokkosCore_PerformanceTest_Mempool ..........   Passed    1.27 sec
```

Reply:
I changed to `alloc_size`. I don't know what other value that I have available here would be a better hint.